tests/cases/dom/traverse/nextAll.js

```diff
@@ -0,0 +1,84 @@
+QUnit.module(
+    "nextAll()",
+    {
+        beforeEach: function ()
+        {
+            document.getElementById("qunit-fixture").innerHTML =
+                '<div class="test-element-nextAll element1"></div>' +
+                '<div class="test-element-nextAll test-class-nextAll element2">' +
+                    '<div class="test-element-nextAll element2-1"></div>' +
+                '</div>' +
+                'Some text' +
+                '<div class="test-element-nextAll test-class-nextAll element3"></div>' +
+                '<div class="test-element-nextAll element4"></div>' +
+                '<div class="test-element-nextAll element5"></div>' +
+                '<div class="test-element-nextAll element6"></div>';
+        }
+    }
+);
+
+
+QUnit.test(
+    "nextAll() with an element in the middle, without selector",
+    function (assert)
+    {
+        const nextAll = mojave.dom.traverse.nextAll;
+        const element = document.querySelector(".test-element-nextAll.element4");
```

Comment:
No reason to use differing grouping classes here, too. By using the same classes, we can more easily just copy & paste the setup code, as needed.